### PR TITLE
added languages.html

### PR DESCRIPTION
**Pull request by ashiscs**
Signed-off-by: ashiscs <ashissingh640@gmail.com>

### CODE REVIEW

File: _includes/languages.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<script type="text/javascript" src="{{site.baseurl}}/assets/js/jquery-3.2.1.min.js"></script>
+<script type="text/javascript" src="{{site.baseurl}}/assets/js/stellarnav.min.js"></script>
+<script type="text/javascript" src="{{site.baseurl}}/assets/js/materialize.min.js"></script>
+
+<script src="{{site.baseurl}}/assets/js/init.js"></script>
+{% include lightbox.html %}
+</body>
+
+
```
Comment (leecalcote) [inline _includes/languages.html]: @ashiscs should we remove these lines?

Comment (ashiscs) [reply _includes/languages.html]: Yes sir @leecalcote this needs to be removed!!

**Comment by ashiscs** [comment]
Ok sir @leecalcote

**Comment by ashiscs** [comment]
Closing this old pr..

**Pull request closed without merging**